File: Back/RecuperApp.Domain/Models/ViewModels/RecivableViewModel.cs
namespace RecuperApp.Domain.Models.ViewModels
{
    public class RecivableViewModel
    {
        public int ShipmentId { get; set; }
        public int ShipmentType { get; set; }
        public string ShipmentTypeName { get; set; }
        public string EmployeeName { get; set; }
        public string CustomerName { get; set; }
        public DateTime Date { get; set; }
        public double TotalPrice { get; set; }
        public List<RecivableDetails> Details { get; set; }

    }
}

File: Front/src/app/models/ViewModel/RecivableViewModel.ts
import { RecivableViewModelDetails as RecivableViewModelDetails } from "./RecivableViewModelDetails";
export class RecivableViewModel {
  ShipmentId: number = -1;
  ShipmentType: number = -1;
  ShipmentTypeName: string = '';
  EmployeeName: string = '';
  CustomerName: string = '';
  Date: string = '';
  TotalPrice: number = -1;
  Details: RecivableViewModelDetails [] = [];
}
